assets/post.css

```diff
@@ -19,10 +22,10 @@ body {
   -moz-font-feature-settings: "kern" 1;
```

Comment:
I'm getting prompted to add `-ms-text-size-adjust:` alongside `-webkit...` when linting, not super familiar with this property...